src/NSwag.Generation.AspNetCore/AspNetCoreOpenApiDocumentGenerator.cs

```diff
@@ -156,13 +156,27 @@ public static JsonSerializerSettings GetSystemTextJsonSettings(IServiceProvider
 
             if (serviceProvider != null)
             {
+#if NETCOREAPP
```

Comment:
If not needed id revert the changes here in this file but keep the tests ok?

Reply:
sure